spec.html

```diff
@@ -8360,21 +8391,20 @@ <h1>[[OwnPropertyKeys]] ( )</h1>
         </emu-alg>
       </emu-clause>
 
-      <emu-clause id="sec-integerindexedobjectcreate" aoid="IntegerIndexedObjectCreate">
-        <h1>IntegerIndexedObjectCreate ( _prototype_, _internalSlotsList_ )</h1>
-        <p>The abstract operation IntegerIndexedObjectCreate with arguments _prototype_ and _internalSlotsList_ is used to specify the creation of new <emu-xref href="#integer-indexed-exotic-object">Integer-Indexed exotic objects</emu-xref>. The argument _internalSlotsList_ is a List of the names of additional internal slots that must be defined as part of the object. IntegerIndexedObjectCreate performs the following steps:</p>
+      <emu-clause id="sec-IntegerIndexedObjectCreate" aoid="IntegerIndexedObjectCreate">
+        <h1>IntegerIndexedObjectCreate ( _prototype_ [ , _additionalInternalSlotsList_ ] )</h1>
+        <p>The abstract operation IntegerIndexedObjectCreate is used to specify the creation of new <emu-xref href="#integer-indexed-exotic-object">Integer-Indexed exotic objects</emu-xref>. The argument _additionalInternalSlotsList_ is a List of the names of additional internal slots that must be defined as part of the object, apart from the ones that are assumed by the Integer-Indexed exotic object internal methods. IntegerIndexedObjectCreate performs the following steps:</p>
         <emu-alg>
-          1. Assert: _internalSlotsList_ contains the names [[ViewedArrayBuffer]], [[ArrayLength]], [[ByteOffset]], and [[TypedArrayName]].
-          1. Let _A_ be a newly created object with an internal slot for each name in _internalSlotsList_.
-          1. Set _A_'s essential internal methods to the default ordinary object definitions specified in <emu-xref href="#sec-ordinary-object-internal-methods-and-internal-slots"></emu-xref>.
+          1. Let _internalSlotsList_ be &laquo; [[Prototype]], [[Extensible]], [[ViewedArrayBuffer]], [[ArrayLength]], [[ByteOffset]], [[TypedArrayName]] &raquo;.
+          1. If _additionalInternalSlotsList_ was given, append each item in it to _internalSlotsList_.
```

Comment:
"append each item in it" -> "append each of its elements"
(use same wording as line 7107)